source/javascripts/controllers/_ProvProfileController.js.erb

```diff
@@ -207,7 +207,7 @@ angular.module("BitriseWorkflowEditor").controller("ProvProfileController", func
 			return "loading";
 		}
 
-		return _.every(viewModel.selectedProvProfile.developerCertificateIdentities, function(anIdentity) {
+		return _.any(viewModel.selectedProvProfile.developerCertificateIdentities, function(anIdentity) {
```

Comment:
No. The status of the "identities" should only be ok if every identity is added.

Reply:
No

Reply:
true